linux-anvil-aarch64/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM continuumio/miniconda3 as builder
+
+RUN conda install constructor
+
+ADD constructors /opt/constructors
+
+RUN cd /opt/constructors/aarch64 && \ 
+    constructor --platform=linux-aarch64 .
+
+FROM arm64v8/centos:7
+
+ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static
+
+LABEL maintainer="conda-forge <conda-forge@googlegroups.com>"
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+# ADD https://now.httpbin.org/when/now /opt/docker/etc/timestamp
+
+# Resolves a nasty NOKEY warning that appears when using yum.
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+# Install basic requirements.
+RUN yum update -y && \
+    yum install -y \
+                   bzip2 \
+                   make \
+                   patch \
+                   sudo \
+                   tar \
+                   which \
+                   libXext-devel \
+                   libXrender-devel \
+                   libSM-devel \
+                   libX11-devel \
+                   mesa-libGL-devel && \
+    yum clean all
+
+# Run common commands
+
+COPY --from=builder /opt/constructors/aarch64/conda_bootstrap-1.0.0-Linux-aarch64.sh miniconda.sh
```

Comment:
Can we do this as a separate project and publish the installer so that users can also use this?

Reply:
https://github.com/jjhelmus/conda4aarch64/releases has a installer built from the packages in the `c4aarch64` channel.  Would probably be a good idea to move it to conda-forge at some time but using that installer initially might be reasonable.  That repository contains the recipes and a few build tools that were used as well.